src/RMLPipeline.Tests/RMLPipeline.Tests.fsproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net9.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
this is an unnecessary change - by all means keep it for testing, but the correct target is 9.0.